### PR TITLE
[翻译完成] flutter.cn/src/docs/cookbook/networking/authenticated-requests.md

### PR DESCRIPTION
**Pull request by XatMassacrE**
翻译完成 authenticated-requests.md
完成 Issue [#48](https://github.com/cfug/flutter.cn/issues/131)

### CODE REVIEW

File: src/docs/cookbook/networking/authenticated-requests.md [hunk 1]
```diff
@@ -34,9 +40,13 @@ Future<http.Response> fetchPost() {
 
 ## Complete example
 
+## 完整示例
+
 This example builds upon the [Fetching Data from the
 Internet](/docs/cookbook/networking/fetch-data/) recipe.
 
+下面的例子基于是[获取网络数据](/docs/cookbook/networking/fetch-data/)中的方法编写的。
```
Comment (chenglu) [inline src/docs/cookbook/networking/authenticated-requests.md]:
```suggestion
下面的例子是基于 [获取网络数据](/docs/cookbook/networking/fetch-data/) 中的方法编写的。
```

File: src/docs/cookbook/networking/authenticated-requests.md [hunk 2]
```diff
@@ -15,12 +15,18 @@ In order to fetch data from many web services, you need to provide
...
+
 ## Add Authorization Headers
 
+## 添加 Authorization Headers
+
 The [`http`]({{site.pub-pkg}}/http) package provides a
 convenient way to add headers to your requests. You can also take advantage of
 the `dart:io` package for common `HttpHeaders`.
 
+[`http`]({{site.pub-pkg}}/http) 包提供了一个非常实用的方法来向请求中添加 headers，你也可以利用 `dart:io` 来使用一些常见的 `HttpHeaders`。
```
Comment (chenglu) [inline src/docs/cookbook/networking/authenticated-requests.md]:
```suggestion
[`http`]({{site.pub-pkg}}/http) 这个 package 提供了相当实用的方法来向请求中添加 headers，你也可以使用 `dart:io` 来使用一些常见的 `HttpHeaders`。
```

File: src/docs/cookbook/networking/authenticated-requests.md [hunk 3]
```diff
@@ -15,12 +15,18 @@ In order to fetch data from many web services, you need to provide
 authorization. There are many ways to do this, but perhaps the most common
 uses the `Authorization` HTTP header.
 
+为了从众多的 web 服务中获取数据，你需要提供相应的授权认证信息。当然了，解决这一问题的方法有很多，而最常见的方法或许就是使用 `Authorization` HTTP header 了。
```
Comment (chenglu) [inline src/docs/cookbook/networking/authenticated-requests.md]:
```suggestion
为了从众多的网络服务中获取数据，你需要提供相应的授权认证信息。当然了，解决这一问题的方法有很多，而最常见的方法或许就是使用 `Authorization` HTTP header 了。
```

**Comment by XatMassacrE** [comment]
@chenglu 修改完毕

**Pull request merged**